stroom-connectors/src/main/java/stroom/connectors/ConnectorProperties.java

```diff
@@ -0,0 +1,55 @@
+package stroom.connectors;
+
+import java.util.Properties;
+
```

Comment:
Provides a wrapper around properties that can be shared with connector libraries, without them having to pull in everything our Stroom Property Service requires.